_posts/2020-03-17-noise-suppression-net-v2.md

```diff
@@ -30,39 +30,104 @@ export class NSNet2 {
     const nn = navigator.ml.getNeuralNetworkContext();
```

Comment:
This example code is outdate, could you please update it to the latest one at https://github.com/webmachinelearning/webnn-samples/blob/master/nsnet2/nsnet2.js#L9?

Reply:
Thanks @Honry, fixed.